src/workers/search/project-search.js

```diff
@@ -3,11 +3,10 @@
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
 // Maybe reuse file search's functions?
-import { isLoaded } from "../../utils/source";
 
 export function findSourceMatches(source, queryText) {
-  const { text } = source;
-  if (!isLoaded(source) || !text || queryText == "") {
+  const text = source.text;
+  if (source.loadedState !== "loaded" || !text || queryText == "") {
```

Comment:
haha - yep. This *does* make it harder to use our utils now that expect SourceRecord :)

but when we go to I Records, it'll be transparent 😉 and we can use the utils again

Reply:
another way to think of this, is maybe we insulate our workers more, so they are more independant